test/test_tests.py

```diff
@@ -8,39 +8,74 @@
 from test.fixtures import tmp_test_dir, click_stub, patch_printfile, config_file, git_access_mock
 from test.test_env import test_env
 
-
-#Testdata contain tuples of (commit_msg, list(calls to exaslct))
-#The goal is to test that for specific commit messages the correct list of calls (with expected arguments) is passed to exaslct
+# Testdata contain tuples of (commit_msg, branch_name, list(calls to exaslct))
+# The goal is to test that for specific commit messages the correct list of calls (with expected arguments) is passed to exaslct
 testdata = [
-    ("last commit", [exaslct_calls.build_call(force_rebuild=False),
-                     exaslct_calls.build_test_container_call(force_rebuild=False),
-                     exaslct_calls.run_db_test_call(),
-                     exaslct_calls.run_db_test_call_for_linker_namespace(),
-                     exaslct_calls.security_scan_call(),
-                     exaslct_calls.push_build_repo_with_sha_call(),
-                     exaslct_calls.push_build_repo_without_sha_call()]
+    #Feature branch should not skip tests if "skip tests" is not in commit message
+    ("last commit", "refs/heads/feature_branch", [exaslct_calls.build_call(force_rebuild=False),
+                                                  exaslct_calls.build_test_container_call(force_rebuild=False),
+                                                  exaslct_calls.run_db_test_call(),
+                                                  exaslct_calls.run_db_test_call_for_linker_namespace(),
+                                                  exaslct_calls.security_scan_call(),
+                                                  exaslct_calls.push_build_repo_with_sha_call(),
+                                                  exaslct_calls.push_build_repo_without_sha_call()]
+     ),
+    #Feature branch should skip tests if "skip tests" is in commit message
+    ("Please be so kind and skip tests!", "refs/heads/feature_branch", [exaslct_calls.build_call(force_rebuild=False),
```

Comment:
We should look for [skip tests] and not plain skip tests

Reply:
Done